frontend/apps/filemanager/filemanager.lua

```diff
@@ -297,8 +298,42 @@ function FileManager:init()
                 }
             },
             -- a little hack to get visual functionality grouping
-            {},
+            {
+            },
         }
+
+        if lfs.attributes(file, "mode") == "file" and string.lower(FrontUtil.getFileNameSuffix(file)) == "sh" then
```

Comment:
More of a general remark, but many shellscripts are just "executables" where whether it's a shell script, a python script or even plain old assembly is just an implementation detail.

Reply:
Yeah, I'd ideally have just chucked a whole load of extensions in DocumentRegistry, and done the check based on the exec bit, but I can't do that because FAT32 :/.

I'll probably add .py to the list, though ;).

Reply:
But the only one we know for sure is executable is a Lua script :-P

Reply:
I'm slightly wary of adding lua to the list because of us being full of lua...